src/snackbar/demo.html

```diff
@@ -0,0 +1,31 @@
+<div class="demo-preview-block">
+<button id="show-toast" type="button">Show Toast</button>
+<button id="show-snackbar" type="button">Show Snackbar</button>
+</div>
+<div class="mdl-js-snackbar"></div>
+<script>
+  'use strict';
```

Comment:
You should never do `'use strict'` in global scope. Wrap this and the following code in an IIFE.